### ISSUE TITLE
Mod breaks LAN server

### ISSUE THREAD

**Issue by SystemlessDev** [RealYusufIsmail-Mc-Mods/TemporalSmith#163]
### General Troubleshooting

- [X] I have checked for similar issues on the Issue-tracker.

### Steps to reproduce

Tested with: 
- TemporalSmith: Aetheric Arsenal Expansion 1.20.1-1.1.1-beta.1
- RealYusufIsmail Core 1.20.1-1.0.3
- Kotlinforforge 4.10.0-all
- Forge 47.2.0

Instructions:
1. Start two clients
2. Use client 1 to make a singleplayer world and open to lan
3. Join with client 2 and see user be kicked with error message

### What is expected?

Lan server works

### What is actually happening?

Client is kicked with the message on the second instance
![image](https://github.com/RealYusufIsmail-Mc-Mods/TemporalSmith/assets/45434286/1f16032e-09a0-4b75-9839-9f61139d7a17)

**Comment by RealYusufIsmail** [comment]
Okay will investigate this issue

**Comment by RealYusufIsmail** [comment]
@SystemlessDev do you mean 1.20.1-1.1.0-beta.1 instead of 1.20.1-1.1.1-beta.1

**Comment by SystemlessDev** [comment]
I did, sorry about that. A little mistype 😅

**Comment by RealYusufIsmail** [comment]
try this jar file and see if it fixes your issue please

https://www.curseforge.com/minecraft/mc-mods/temporalsmith/files/all?page=1&pageSize=20

**Comment by SystemlessDev** [comment]
Issue still occurs with 1.1.0-beta.2

**Comment by RealYusufIsmail** [comment]
<img width="1001" alt="Screenshot 2024-01-21 at 08 57 49" src="https://github.com/RealYusufIsmail-Mc-Mods/TemporalSmith/assets/67903886/cb0fc2b0-22fe-4e62-bd9c-b5b0a6dbfbc0">

I ran LAN on my game with the my mod and did not recieve this error. I dont think this error is from my mod. Are you running any other mods

**Comment by SystemlessDev** [comment]
No, I am not running any other mods than stated in the original issue.

Debug log server: https://paste.essential.gg/CVFjNWT3Kf
Debug log client: https://paste.essential.gg/giAm2JH64c

**Comment by RealYusufIsmail** [comment]
> No, I am not running any other mods than the ones listed in the original issue.
> 
> 
> 
> Here's logs from a test session with only your mods. Sadly Minecraft doesn't log this very well.
> 
> Debug log server: https://paste.essential.gg/CVFjNWT3Kf
> 
> Debug log client: https://paste.essential.gg/giAm2JH64c
> 
> 
> 
> Essential makes Minecraft log the exception caught. Here's a copy of that exception. 
> 
> https://paste.essential.gg/inuGzR7YHK

Will check the logs out and seee what the issue is.

Sorry for the issues.
